deploy/compose/README.md

```diff
@@ -71,3 +147,9 @@ COMPOSE_PROFILES=hoprd,admin-ui,metrics,metrics-vis docker compose up -d
 ```
 
 The same list of `COMPOSE_PROFILES` should be supplied for the `docker compose down` command.
+
+6. Stop only the hopr node
+
+```shell
+COMPOSE_PROFILES=hoprd docker compose down
```

Comment:
I would mention somewhere that the down command needs to be executed with the same `COMPOSE_PROFILES=....` that was used in the startup command, otherwise it won't stop the containers.